junit-jupiter-engine/src/test/java/org/junit/jupiter/engine/execution/ParameterResolutionUtilsTests.java

```diff
@@ -126,9 +125,9 @@ void resolveMultipleArguments() {
 			}
 		}));
 
-		invokeMethod();
+		Object[] arguments = resolveMethodParameters();
 
-		verify(instance).multipleParameters("0", 1, 2.0);
+		assertThat(arguments).hasSize(3).containsExactly("0", 1, 2.0);
```

Comment:
```suggestion
		assertThat(arguments).containsExactly("0", 1, 2.0);
```

`hasSize()` is already covered by `containsExactly()`